README.md

```diff
@@ -61,25 +61,28 @@ The manager looked at the engineer and was not happy.
 
 UPDATE: A fully encrypted laptop of our senior developer has been stolen from his bag while he was on a train, returning from a conference in late 2017. Spooky.
 
-### Encryption
+### Storage Encryption and Secure Work Environment
 
-All data related to the company and clients must be either stored in the cloud (Google Drive/Docs/Sheets in the company domain) or on encrypted media (encrypted volume or drive), unless the data in question is worthless or public.
+All data pertaining to the company and its clients must be either stored in the cloud (specifically, Google Drive/Docs/Sheets within the company domain) or on encrypted media (an encrypted volume or drive).
+This excludes data categorized as being of no value or already publicly available.
 
-EDIT: We are getting a lot of questions about who this applies to, so let's make it clear:
-everyone except:
+In addition, Developers roles (after 1nd stage of Trial period i.e. finishing Trial task#3) are required to set up a Secure Work Environment on their workstations.
```

Comment:
right now our guide mentions word Developer more often than Engineer;
do you think it will be more clear If I change it to 

this affects following roles:
Staff Python Backend Engineer.
Senior Python Backend Engineer

but I guess just saying Engineers' roles will be even more confusing,
so I will change "Developers roles" to just "developers"
```suggestion
In addition, developers (after 2nd stage of Trial period i.e. that are about to start working on our clients' code base) are required to set up a Secure Work Environment on their workstations.
```